designs/developer/add-45/Pipelines-update/Pipelines-update.md

```diff
@@ -2,18 +2,15 @@
 parent: Developer
 version: 4.5
 ---
-# Add page- Updates
+# Pipelines section in Add Flows (Updates in 4.5)
 
```

Comment:
I think we should consider moving this file (Pipelines_update.md) to the folder called sections (and getting rid of the Piplines-update folder). If you look in all the add-* directories, they have a sections folder. All the sections of the Add flow are kept there. Also, I am wondering how this page fits in with the existing page openshift-origin-design/designs/developer/add-43/sections/pipelines/pipelines.md? There is relevant info there that is not copied over. We can discuss in our meeting!